picard/ui/coverartbox.py

```diff
@@ -135,7 +135,7 @@ def dropEvent(self, event):
             event.acceptProposedAction()
 
     def scaled(self, *dimensions):
-        return (self.pixel_ratio * dimension for dimension in dimensions)
+        return (int(self.pixel_ratio * dimension) for dimension in dimensions)
```

Comment:
> I have applied the proposed change (but I am just calling `round`, no need for `int` since _The return value is an integer if ndigits is omitted or None_).

Actually you're just calling `int()`, I guess `round()` is preferable in this case.

Also, yes, no need of `int(round())` with Python >= 3.6.

It wasn't always the case, hence my error:

```
zas:~$ python2
Python 2.7.18 (default, Mar  8 2021, 13:02:45) 
[GCC 9.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> type(round(0.5))
<type 'float'>
>>> 
zas:~$ python3
Python 3.8.10 (default, Jun  2 2021, 10:49:15) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> type(round(0.5))
<class 'int'>
```

Reply:
Yeah sorry about that, I'd forgotten to push my local changes, thanks for the merge :)